diem-unified-charter.md

```diff
@@ -1,12 +1,25 @@
 # Introduction
 
-Digital Emblems provide a mechanism for conveying a set of attributes
-bound to an asset that is presented to a validator and
-asserted by an issuer.
+An emblem is a device, symbol, or figure adopted and used as an identifying mark.
+As such, emblems provide a mechanism for conveying a set of attributes
+bound to an asset. An emblem may be presented to a validator and attributes are asserted by an issuer.
+
+In culture, emblems such as a flag, badge or coat of arms communicate group identity.
+In speech, emblems are specific nonverbal gestures or signals that have a direct verbal translation and are widely understood within a particular culture or community.
+The emblems of the International Committee of the Red Cross (ICRC)  are symbols of protection under the Geneva Conventions and are to be worn by all medical and humanitarian personnel and also displayed on their vehicles and buildings while they are in an active warzone, and all military forces operating in an active warzone must not attack entities displaying these emblems.
+ISO 7010 defines a set of emblems that can be used to identify hazards, these include the skull and crossbones for toxic material, the ionizing radiation symbol and the biological hazard symbol.
+Today often these emblems/symbols require a sense of sight, or touch to become known to the receiver.
```

Comment:
I cut Orie's text a bit shorter here and removed some of the more detailed examples. I think these examples are already good enough but we could of course re-add them if anybody feels specifically those examples were important.